sample/src/main/java/com/microsoft/gctoolkit/sample/collections/XYDataSet.java

```diff
@@ -1,24 +1,21 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
-package com.microsoft.gctoolkit.collections;
+package com.microsoft.gctoolkit.sample.collections;
```

Comment:
Am I to understand from this that this PR would move XYDataSet to the sample module? As it stands, XYDataSet is a class that is core to the API.

Reply:
It certainly does not seem to have anything to do with core, as it would be for displaying datasets. IMHO it would be better out of core.